### PR TITLE
Fix channel name autocomplete

### LINKED ISSUE #8480: autocompletion chat for addressing channels fails
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Address whole Talk channel via chat using the @{name_channel}
2. just type in @ and type in channel name
3. it will not autocomplete

### Expected behaviour

I expected an autocompletion for that channel but it fails to do so. 

### Actual behaviour
The @{name_channel} does not turn into a valid chat @ handle

## Talk app

**Talk app version:** (see apps admin page: `/index.php/settings/apps`)

Running on current Nextcloud NC instance (2022-12-21)

**Custom Signaling server configured:** yes/no and version (see additional admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

_Unable to mention_

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#turn_server`)
_Unable to mention_

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#stun_server`)
_Unable to mention_

## Browser

**Microphone available:** yes

**Camera available:** yes

**Operating system:** Windows/Ubuntu/...

Apple Mac OS X

**Browser name:** Firefox/Chrome/...

Google Chrome	

**Browser version:** 85/96/...

108.0.5359.124 (Official Build) (arm64)

### Browser log

Full browser log below 

<details>
```
/apps/spreed/:1 Refused to apply style from 'https://cloud.nextcloud.com/apps/tasks/css/tasks-talk.css?v=46da7874-1762' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
jquery-migrate.min.js:2 JQMIGRATE: Migrate is installed, version 3.4.0
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.0
e @ index.js:19
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
ProxyBus.ts:11 Proxying an event bus of version 2.1.1 with 3.0.2
e @ ProxyBus.ts:11
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
ConsoleLogger.js:60 [WARN] viewer: Some mimes were ignored because they are not enabled in the server previews config Object
value @ ConsoleLogger.js:60
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.esm.js:11
viewer.js:43 Uncaught TypeError: Cannot read properties of undefined (reading 'fileActions')
    at viewer.js:63:13
    at viewer.js:65:2
    at viewer.js:65:2
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.js:19
globals.js:60 oc_appswebroots is deprecated: use OC.appswebroots instead, this will be removed in Nextcloud 20
tn @ globals.js:60
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
l @ index.esm.js:11
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.js:19
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
c @ index.esm.js:11
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.js:19
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
u @ index.esm.js:11
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
u @ index.esm.js:11
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.js:19
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
c @ index.esm.js:11
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
u @ index.esm.js:11
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
e @ index.js:19
ProxyBus.ts:16 Proxying an event bus of version 2.1.1 with 3.0.2
e @ ProxyBus.ts:16
ProxyBus.ts:16 Proxying an event bus of version 2.1.1 with 3.0.2
e @ ProxyBus.ts:16
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
major.js:2 Proxying an event bus of version 2.1.1 with 1.3.0
e @ major.js:2
browserCheck.js:29 Detected browser Chrome 108 (108.0.0.0)
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
m @ index.esm.js:11
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
t @ index.js:19
ProxyBus.ts:16 Proxying an event bus of version 2.1.1 with 3.0.2
e @ ProxyBus.ts:16
ProxyBus.ts:16 Proxying an event bus of version 2.1.1 with 3.0.2
e @ ProxyBus.ts:16
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.2
t @ index.js:19
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
t @ index.es.js:2337
index.esm.js:11 Proxying an event bus of version 2.1.1 with 3.0.2
l @ index.esm.js:11
index.es.js:2337 Proxying an event bus of version 2.1.1 with 1.3.0
e @ index.es.js:2337
index.js:19 Proxying an event bus of version 2.1.1 with 3.0.0
e @ index.js:19
session-heartbeat.js:103 session heartbeat polling started
init-collections.js:42 OCA.Sharing.ShareSearch not ready
(anonymous) @ init-collections.js:42
init-collections.js:42 OCA.Sharing.ShareSearch not ready
(anonymous) @ init-collections.js:42
tflite-simd.js:9 [WASM] Loading model of size: 249792
tflite-simd.js:9 INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
printChar @ tflite-simd.js:9
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 oc_defaults is deprecated: use OC.theme instead, this will be removed in Nextcloud 20
tn @ globals.js:60
NcActions.js:2 You need to fill either the text or the ariaLabel props in the button component. Object o
mounted @ NcActions.js:2
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
signaling.js:1081 Not connected to signaling server yet, defer joining room 3536879508
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
MessagesList.vue:785 First visible message not found:  null
updateReadMarkerPosition @ MessagesList.vue:785
MessagesList.vue:860 Message to focus not found in DOM 1598671
focusMessage @ MessagesList.vue:860
core.js:1003 Deprecated as of 10.7.0. highlightBlock will be removed entirely in v12.0
core.js:1003 Deprecated as of 10.7.0. Please use highlightElement now.
MessagesList.vue:860 Message to focus not found in DOM 1597755
focusMessage @ MessagesList.vue:860
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:323 OC.requestToken changed GzrzeMnyPancKl9c/KWVY2l8YnjGe902RxJq1pu14Cg=:fHmGPayWZf26Tic3qtKgMCoQEUiuT5VnI3Aem+GHl30=
/ocs/v2.php/apps/user_status/api/v1/statuses/abdourahamane.boinaidi%40infomaniak.com:1          Failed to load resource: the server responded with a status of 404 ()
/ocs/v2.php/apps/user_status/api/v1/statuses/aleyna.arslan557%40gmail.com:1          Failed to load resource: the server responded with a status of 404 ()
/ocs/v2.php/apps/user_status/api/v1/statuses/brennamccullough6%40gmail.com:1          Failed to load resource: the server responded with a status of 404 ()
/ocs/v2.php/apps/user_status/api/v1/statuses/Automated_Availability_Tests:1          Failed to load resource: the server responded with a status of 404 ()
/ocs/v2.php/apps/user_status/api/v1/statuses/curtis%40mashedkeyboard.me:1          Failed to load resource: the server responded with a status of 404 ()
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
/ocs/v2.php/apps/user_status/api/v1/statuses/hp%40fossasia.org:1          Failed to load resource: the server responded with a status of 404 ()
/ocs/v2.php/apps/user_status/api/v1/statuses/lahbibe.saadi%40tasgroup.eu:1          Failed to load resource: the server responded with a status of 404 ()
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
index.js:323 OC.requestToken changed hKWmeUlLs+7f1XMneYXfUi2kXpM8MV23UDSO9vpgy+c=:4+bTPCwv67q5sQtML/LqAW7ILaNUBRXmNFb6u4BSvLI=
20221221_092637.gif:1          Failed to load resource: the server responded with a status of 404 ()
xhr.js:164 Refused to set unsafe header "Content-Length"
(anonymous) @ xhr.js:164
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.js:323 OC.requestToken changed 58Zgl7bAiECVczp83Z8w69iP1THD4Xt19rpN1tboZAs=:gIUV0tOk0BTzF0IXi+gFuJvjpgGr1TMkktg5m6zaE14=
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.js:323 OC.requestToken changed 8dzW/13NmlyFmIGIf2WPSKEY8vr018Dkxuhacwg85vo=:lp+jujipwgjj/PnjKRK6G+J0gcqc44i1ooouPnIOka8=
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.js:323 OC.requestToken changed ZkWRdpOX8og+358K24yEh0kl4h/SM0qw/JugHxYTQYg=:AQbkM/bzqtxYu+dhjfux1ApJkS+6BwLhmPnUUmwhNt0=
injected.js:17 [USO] Button detection aborting due to timeout.
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
/core/preview?fileId=6568287&x=-1&y=768&a=1:1          Failed to load resource: the server responded with a status of 404 ()
index.js:987 Widget for rich object type call not registered
(anonymous) @ index.js:987
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
tn @ globals.js:60
index.js:987 Widget for rich object type call not registered
(anonymous) @ index.js:987
/core/preview?fileId=6568287&x=-1&y=768&a=1:1          Failed to load resource: the server responded with a status of 404 ()
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
index.js:987 Widget for rich object type file not registered
(anonymous) @ index.js:987
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
index.js:323 OC.requestToken changed JIwftentZM0J0n5VWXQkvNT68gUjprCvu48cocGZ4yo=:Q89q8IyJPJlvtgY+DwMR75eWgTVLkvj+3+1o7LurlH8=
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
20221221_101636.gif:1          Failed to load resource: the server responded with a status of 404 ()
xhr.js:164 Refused to set unsafe header "Content-Length"
(anonymous) @ xhr.js:164
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
participantsStore.js:359 Participant not found for conversation wejc2q9q Object
updateUser @ participantsStore.js:359
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
20221221_101903.gif:1          Failed to load resource: the server responded with a status of 404 ()
xhr.js:164 Refused to set unsafe header "Content-Length"
(anonymous) @ xhr.js:164
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
index.js:323 OC.requestToken changed YSQPZ9xYobja4hFcyB4BSrXmpcs5FmUW1Mw5le+jOig=:Bmd6Irk8+ey8hmk3nmk0GfaK1vtRIi1HsK5N2JWRTX0=
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
injected.js:17 [USO] Button detection aborting due to timeout.
focus-trap.esm.js:328 Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at p (focus-trap.esm.js:328:13)
    at Object.activate (focus-trap.esm.js:590:9)
    at NcPopover.js:1:12332
    at c (dist.js:1:80785)
    at Generator._invoke (dist.js:1:80573)
    at Generator.next (dist.js:1:81214)
    at o (NcPopover.js:1:11473)
    at s (NcPopover.js:1:12486)
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
injected.js:17 [USO] Button detection aborting due to timeout.
wejc2q9q:1 Refused to apply style from 'https://cloud.nextcloud.com/apps/tasks/css/tasks-talk.css?v=46da7874-1762' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
index.js:323 OC.requestToken changed 1x8X9UB041J7AJxZFLqPjwifoQQGny/UcPRkzmS5Ns8=:sFxisCUQuwYdZOQyQs263Evz0jRuq2eFFJYQgx6LQZo=
```

</details>

## Server configuration

As and enduser the server configuration details are unknown to me. Dropping these question for this bug report. 

## Additional information 

As mentioned by @st3iny it could be caused by the use of emoji for chat channel names. And @nickvergessen replied the following :

_Oh... we do stripos === 0_
_instead of !== false_

### PR DESCRIPTION
**Pull request by st3iny**
Fix #8480

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Review by vitormattos — CHANGES_REQUESTED** [review]
Will be good cover this case on unit tests:
https://github.com/nextcloud/spreed/blob/master/tests/php/Chat/ChatManagerTest.php#L664-L684

**Comment by nickvergessen** [comment]
@vitormattos please take over.

File: tests/php/Chat/ChatManagerTest.php [hunk 1]
```diff
@@ -663,15 +663,36 @@ public function testClearHistory(): void {
...
+			'found a in all' => [
+				'a', 'room', true
+			],
+			'found h in here' => [
+				'h', 'room', true
+			],
+			'case sensitive, not found A in all' => [
+				'A', 'room', false
+			],
+			'case sensitive, not found A in here' => [
```
Comment (nickvergessen) [inline tests/php/Chat/ChatManagerTest.php:675]:
```suggestion
			'case sensitive, not found H in here' => [
```

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**